deployment/terraform/assets/dashboard_data.json

```diff
@@ -657,18 +657,15 @@
           "format": "time_series",
           "intervalFactor": 1,
           "legendFormat": "{{instance}}",
-          "metric": "mattermost_api_time_sum",
+          "metric": "mattermost_http_request_duration_seconds_sum",
```

Comment:
This should be api_time_sum

Reply:
Not sure how that gets updated from Grafana. The query uses the correct metric.

Reply:
What do you mean "updated from Grafana" ? I meant that both api_time_sum and http_request_duration_seconds sum capture the same thing. I have a PR out which removes the http_request_duration metric from Prometheus. We should just be using api_time_sum.

Reply:
I mean, the query uses the right metric but that's not reflected in that JSON field. I don't know where I can set that from Grafana UI.

Reply:
That's because that metric [is going to be removed](https://github.com/mattermost/mattermost-server/pull/14520)

Reply:
Updated anyway :P